lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -871,6 +873,14 @@ def _post(self, data):
         url = reverse('verify_student_create_order')
         return self.client.post(url, data)
 
+    def _check_payment_data(self, response, expected_form_data):
+        """ DRY helper. """
+        data = json.loads(response.content)
+        self.assertEqual(set(data.keys()), PAYMENT_DATA_KEYS)
+        payment_form_data = data['payment_form_data']
```

Comment:
as discussed offline, the 'payment_data' dictionary is the whole of the response from this endpoint, so this doesn't need to be changed.